packages/synapse-react-client/test/containers/AccessRequirementList/DataAccessRequestAccessorsFilesForm.test.tsx

```diff
@@ -78,7 +78,11 @@ const mockGetDataRequestForUpdate = jest.spyOn(
 
 const mockUpdateDataAccessRequest = jest
   .spyOn(SynapseClient, 'updateDataAccessRequest')
-  .mockImplementation(req => Promise.resolve(req))
+  .mockImplementation(req => {
+    // Update the 'GET' mock to return the updated object
+    mockGetDataRequestForUpdate.mockResolvedValue(req)
+    return Promise.resolve(req)
+  })
```

Comment:
Since we're now pushing to and pulling from the server on every change, the mocks need to be constantly updated